js/controller.js

```diff
@@ -335,18 +338,27 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
      * @description
      * Should be used with deviceorientation event listener.
      * Uses for video 360 on mobile devices for setting necessary coordinates (relevant with start device orientation)
+     * Before playing this.vrMobileOrientationChecked is equal false,
+     * if need to check value for device orientation set this.checkDeviceOrientation = true
      * @param {object} e The event object
      */
     handleVrMobileOrientation: function(e) {
-      if (!this.vrMobileOrientationChecked) {
+      if (!this.vrMobileOrientationChecked || this.checkDeviceOrientation) {
```

Comment:
Both of these involve "checking" while the function name uses "handle." Should the function be renamed to checkVrMobileOrientation?